### ISSUE TITLE
Research and implement a bug logging service for DDEV-UI

### ISSUE THREAD

**Issue by dclear** [drud/ddev-ui#139]
## What happened (or feature request):

Stub: Research and implement a bug logging service for the UI. These are tools for tracking tracking / catching errors on released code / running installs. 

Options:
- [ ] https://rollbar.com/
- [ ] https://airbrake.io/
- [ ] http://jslogger.com/
- [ ] https://www.bugsnag.com/

**Comment by dclear** [comment]
The next step here is a tool needs to be agreed up and picked. CCing @mafriend for input.

**Comment by mafriend** [comment]
Cool! That's kind of what I thought 😀 I'm reading up on some of the bug loggers mentioned above and so far I know for sure bugsnag is GDPR compliant and does a good job making sure privacy settings are respected. Sentry is also compliant! And they have pretty bomb reporting!

**Comment by alkymst** [comment]
No, I found npm packages for sentry with a react app. It's easy to setup and works with the rest of what we have going on. So I'm totally good with Sentry.

**Comment by mafriend** [comment]
Dope! I'm very cool with that!
